charts/datadog/templates/agent-clusterchecks-deployment.yaml

```diff
@@ -125,6 +125,7 @@ spec:
 {{- end }}
 {{- end }}
         env:
+          {{- include "components-common-env" . | nindent 10 }}
```

Comment:
Some of the ENVs defined below are part of `components-comment-env`, so now they're redundant. We should check that the values are the same before deleting them.

Reply:
Right. Updated the PR